### PR TITLE
Fix acceptance testing

### PR DESCRIPTION
**Pull request by marinsalinas**

### CODE REVIEW

File: mongodbatlas/resource_mongodbatlas_cluster.go [hunk 1]
```diff
@@ -734,7 +734,19 @@ func flattenRegionsConfig(regionsConfig map[string]matlas.RegionsConfig) []map[s
 func resourceClusterRefreshFunc(name, projectID string, client *matlas.Client) resource.StateRefreshFunc {
 	return func() (interface{}, string, error) {
 		c, resp, err := client.Clusters.Get(context.Background(), projectID, name)
-		if err != nil {
+
+		log.Printf("LOG C----------------------------\n\n %+v \n\n ----------------------------", c)
+		log.Printf("LOG R----------------------------\n\n %+v \n\n ----------------------------", resp)
+		log.Printf("LOG E----------------------------\n\n %+v \n\n ----------------------------", err)
```
Comment (PacoDw) [inline mongodbatlas/resource_mongodbatlas_cluster.go]: Remove the logs 👍

File: mongodbatlas/resource_mongodbatlas_cloud_provider_snapshot.go [hunk 2]
```diff
@@ -184,7 +184,14 @@ func resourceCloudProviderSnapshotRefreshFunc(requestParameters *matlas.Snapshot
 	return func() (interface{}, string, error) {
 		c, resp, err := client.CloudProviderSnapshots.GetOneCloudProviderSnapshot(context.Background(), requestParameters)
 
-		if err != nil || c.Status == "failed" {
+		// log.Printf("C --------------------------- \n\n %+v \n\n", c)
+		// log.Printf("R --------------------------- \n\n %+v \n\n", resp)
+		// log.Printf("E --------------------------- \n\n %s \n\n", err)
```
Comment (PacoDw) [inline mongodbatlas/resource_mongodbatlas_cloud_provider_snapshot.go]: Remove the comments as well 👍

Comment (marinsalinas) [reply mongodbatlas/resource_mongodbatlas_cloud_provider_snapshot.go]: Done!

File: mongodbatlas/resource_mongodbatlas_cluster.go (hunk 1 above)
Comment (marinsalinas) [reply mongodbatlas/resource_mongodbatlas_cluster.go]: Done!

**Pull request merged**